app/javascript/packs/mock-device-profiling.tsx

```diff
@@ -0,0 +1,83 @@
+import { render } from 'react-dom';
+import { useInstanceId } from '@18f/identity-react-hooks';
+import { ChangeEvent } from 'react';
+
+/**
+ * Loads the ?session_id=SESSION_ID param from the <script> tag. Unfortunately
+ * import.meta.url doesn't have live URL params so we need to scrape the DOM.
+ */
+function loadSessionId(): string | undefined {
+  let sessionId;
+  Array.from(document.scripts).every((scriptTag) => {
+    if (scriptTag.src.includes('session_id')) {
+      sessionId = new URL(scriptTag.src).searchParams.get('session_id');
+      return false;
+    }
+    return true;
+  });
+  return sessionId;
+}
+
+function submitMockFraudResult({ result, sessionId }: { result: string; sessionId?: string }) {
+  window.fetch('/test/device_profiling', {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({
+      session_id: sessionId,
+      result,
+    }),
+  });
+}
+
+function mockFraudResultSelected(event: ChangeEvent) {
+  if (!(event.target instanceof HTMLSelectElement)) {
+    return;
+  }
+
+  const { value } = event.target;
+
+  if (value === 'chaotic') {
+    document.body.style.transform = 'rotate(180deg)';
```

Comment:
I would like to have more easter eggs to show what we can do with JS like this. My first idea was to slowly rotate the whole page... but this was faster. I am open to removing if anybody feels that's better.

Reply:
I added some more options in a7e8f757f 👹